backend/back/src/types/jobParams.ts

```diff
@@ -223,3 +233,28 @@ export class FacebookPageParams {
 	@IsString()
 	pageId: string;
 }
+
+enum RiotRegion {
+	"br",
+	"eun",
+	"euw",
+	"jp",
+	"kr",
+	"la",
+	"la2",
+	"na",
+	"oc",
+	"tr",
+	"ru",
+}
+type RiotRegionType = keyof typeof RiotRegion;
+
+export class RiotActionsParams extends UniqueJobParams {
+	@IsEnum(RiotRegion)
+	region: RiotRegionType;
+
+	@IsString()
+	@MinLength(3)
+	@MaxLength(16)
+	summoner: string;
```

Comment:
Don't you have a list o sommner ? Could ask pablo for that

Reply:
Summoner = a player (here I want the player username)
Champion = a playable character